cocos/scene-graph/node.ts

```diff
@@ -1920,8 +1937,13 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
      * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
      */
     public _onPostActivated (active: boolean): void {
+        if (this._eventMask & ACTIVE_ON) {
```

Comment:
Is post active the same as node active?

Reply:
Yes, u could refer the code in `node-activator.ts`.
![WeChat0ad0a025b8fa1547a1cf0db50b81c382](https://github.com/cocos/cocos-engine/assets/493372/268ba367-844a-4021-9461-d3d715b2939d)

Reply:
Also you could refer which in Creator 2.x ( https://github.com/cocos/cocos-engine/blob/cedd59c725a76736295773586973e3360d5eaa17/cocos2d/core/CCNode.js#L1747 )

2.x also did ActionManager resume/pause in _onPostActivated

Reply:
Ref: https://github.com/cocos/cocos-engine/pull/17141